### ISSUE TITLE
NaN values in the results 

### ISSUE THREAD

**Issue by hfawaz** [tcapelle/TimeSeries_fastai#1]
Hi Thomas, 

Thanks for sharing this! 

I am just wondering about the NaN values in the results, can you explain ? 
We are currently in the process of gathering the results on these 128 datasets, using [our own implementations in Keras](https://github.com/hfawaz/dl-4-tsc) and would like to compare the accuracy with yours. 

Thanks again ;)

**Comment by tcapelle** [comment]
I will check this next week. If you look at the code I put a lazy try/except.

**Comment by tcapelle** [comment]
Ok, it is NonInvasiveFetalECGThorax1  not FATAL! that was one problem. I had some issues with batch_size=1 and BatchNorm (it is a pytorch issue), so it is fixed now.
